news/2018-05-16-performance-updates.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: "New Scoring Updates: osu! and osu!mania"
+date: 2018-05-16 14:00:00 +0000
+---
+
+A brand new set of pp adjustments have made their way through the pipeline for osu! and osu!mania – the first of their kind in several years! Wonder how the changes might have affected you and your ranks? Read on!
+
+![](https://assets.ppy.sh/media/generic-header.png)
+
+Over the past few weeks, much forward progress has been made on the [performance processor](https://github.com/ppy/osu-performance) (mainly thanks to [Tom94](https://github.com/Tom94)). As a result, we have also been able to quickly iterate over community proposals and begin breathing life back into the performance and star rating algorithms to keep them relevant.
+
+**Note that as these changes are applied, ranks may fly around a bit.** We are doing our best to alleviate the visual effect of reprocessing, but as this is the first time we've done this kind of operation in a while it is a learning experience – there's a lot of data to process!
+
+We begin this journey with the following changes:
+
+## inactive players
+
+Until recently, we have not been marking inactive players (no score recorded in 3 months) as such in all cases, which means that as we perform this reprocessing, **all users will generally see a sudden increase in rank**.
+
+## osu!
+
+[Adjusting Hidden Mod As Speed Multiplier](https://github.com/ppy/osu-performance/pull/42)
+
+[AR Fix for aim on HD](https://github.com/ppy/osu-performance/pull/47) to fix further [issues raised by NoMoreDiv](https://github.com/ppy/osu-performance/issues/45).
+
+## osu!mania
+
+Proposed by [Shoegazer](https://osu.ppy.sh/users/2520707) as a "bandaid" fix and [implemented by MillhioreF(https://github.com/ppy/osu-performance/pull/15), osu!mania has received a set of changes that adjust its performance calculations quite drastically.
```

Comment:
An ] is missing > `[implemented by MillhioreF](https://github.com/ppy/osu-performance/pull/15)`